ansible_collections/arista/avd/roles/eos_designs/python_modules/overlay/router_bgp.py

```diff
@@ -122,7 +122,7 @@ def _peer_groups(self) -> list | None:
                 peer_group_config = {"remote_as": self.shared_utils.bgp_as}
                 if self.shared_utils.wan_role:
                     # WAN OVERLAY peer group
-                    # TODO Add TTL max hop to the peer group on the Pathfinder once agreed upon
+                    peer_group_config["ttl_maximum_hops"] = 1
```

Comment:
let's do the same for the RR_OVERLAY_PEERS peer-group when the router is a WAN router using DPS to peer. Put this behind a guard checking for WAN role and I will track an issue to separate the RR_OVERLAY_PEERS and the WAN_RR_PEERS in a future PR.